src/login/logind-action.c

```diff
@@ -299,12 +300,49 @@ static int handle_action_sleep_execute(
         return handle_action_execute(m, handle, ignore_inhibited, is_edge);
 }
 
+static int manager_handle_action_secure_attention_key(
+                Manager *m,
+                bool is_edge,
+                char *seat) {
+        int r;
+        Seat *o;
+        _cleanup_free_ char *p = NULL;
+
+        assert(m);
+        assert(seat);
```

Comment:
not all buttons have seats assigned. please handle gracefully if seat is NULL here.

i.e. just return with zero early in that case.